<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TestPortClass</class>
 <widget class="QDialog" name="TestPortClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TestPort</string>
  </property>
  <widget class="QPlainTextEdit" name="plainTextEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>581</width>
     <height>331</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonStart">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>350</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Start  Test</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonStop">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>350</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Stop Test</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonEStop">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>350</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>E Stop Test</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="testport.qrc"/>
 </resources>
 <connections/>
</ui>
